pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -550,6 +552,19 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 	return reconcile.Result{RequeueAfter: ResyncPeriod}, nil
 }
 
+func updateIPsecTelemetry(newOperConfigSpec *operv1.NetworkSpec) {
+	if newOperConfigSpec == nil || newOperConfigSpec.DefaultNetwork.OVNKubernetesConfig == nil {
```

Comment:
what if the user switches to a new CNI from ovn-k? the metrics will still report old values indicating ipsec is enabled. You should reset the metric for this scenario.